script/Arcueid/Day 8/08_00_ARC08_3_2.txt

```diff
@@ -398,7 +398,7 @@ Chattering away in a voice full of naive exuberance,
 [sha:6c3da51b47a6ff56b6289450b7b01e77cae155ea]{
 -- Page 222, Offset 20221.
 -- ……クソ、なんだこの吸血鬼……ずるいぞ……さっきまで叱りつける気満々だったのに、そんな顔をされたら気持ちが萎えてしまうというか……。
-...Shit. What's up with this vampire... That's not fair...
+...Shit. What's up with this vampire...? That's not fair...
```

Comment:
I generally think of `?` more as an inflection signifier than anything else, and for me it doesn't really fit with how I read this (both sentences trailing off, descending)

Reply:
I agree with ross